### PR TITLE
Enable Bootstrap in project#show for development/test

### PR DESCRIPTION
**Pull request by dmarcoux**
Following up on our discussions in the standup.

### CODE REVIEW

**Comment by dmarcoux** [comment]
@openSUSE/obs-frontend Please have a look.

File: src/api/app/controllers/webui/project_controller.rb [hunk 1]
```diff
@@ -206,10 +206,10 @@ def show
     @comment = Comment.new
     render :show, status: params[:nextstatus] if params[:nextstatus]
 
-    # TODO: Remove the `return unless` and the flash once this should be available to all beta users
-    return unless User.current && User.current.in_beta? && (User.current.is_admin? || User.current.is_staff?)
+    # TODO: Remove the `return unless` and the flash once this should be available to all beta users on all environments
+    return unless User.current && User.current.in_beta? && (Rails.env.development? || Rails.env.test?)
```
Comment (Ana06) [inline src/api/app/controllers/webui/project_controller.rb]: why do we need this for test?

Comment (ChrisBr) [reply src/api/app/controllers/webui/project_controller.rb]: Because we should test it :thinking:

Comment (dmarcoux) [reply src/api/app/controllers/webui/project_controller.rb]: We will write some feature specs once we start to have a bit more content in the Bootstrap version of project pages.

Comment (Ana06) [reply src/api/app/controllers/webui/project_controller.rb]: until now there are no tests and we don't test the views

Comment (ChrisBr) [reply src/api/app/controllers/webui/project_controller.rb]: But we test workflows and for that you need the view :smile:

Comment (Ana06) [reply src/api/app/controllers/webui/project_controller.rb]: I would add it when it is necessary. No test need this at the moment. But I have no strong opinion

**Comment by dmarcoux** [comment]
Someone to approve or request changes... please?

**Pull request merged**